app/serializers/applicant_serializer.rb

```diff
@@ -1,5 +1,3 @@
-class ApplicantSerializer
-  include FastJsonapi::ObjectSerializer
-  attribute :first_name, :last_name, :email_address
-  attribute :date_of_birth
+class ApplicantSerializer < ActiveModel::Serializer
+  attributes :first_name, :last_name, :email_address, :date_of_birth
```

Comment:
Should National Insurance number be included here now too? (Just a reminder - I'm aware that you haven't rebased so might not have those changes yet.)